docs/book/component-gallery/orchestrators/gcloud-vertexai.md

```diff
@@ -68,6 +74,44 @@ You can now run any ZenML pipeline using the Vertex orchestrator:
 python file_that_runs_a_zenml_pipeline.py
 ```
 
+### Run pipelines on a schedule
+
+The Vertex Pipelines orchestrator supports running pipelines on a schedule, using
+logic resembling the [official approach recommended by GCP](https://cloud.google.com/vertex-ai/docs/pipelines/schedule-cloud-scheduler).
+
+ZenML utilizes the [Cloud Scheduler](https://cloud.google.com/scheduler) and
+[Cloud Functions](https://cloud.google.com/functions) services to enable scheduling
+on Vertex Pipelines. The following is the sequence of events that happen when running
+a pipeline on Vertex with a schedule:
+
+* Docker image is created and pushed (see above [containerization](../../advanced-guide/pipelines/containerization.md)).
+* Cloud Function created that creates the Vertex Pipeline job when triggered.
+* Cloud Scheduler job is created that triggers the Cloud Function on the defined schedule.
+
+Therefore, to run on a schedule, the runner environment needs permissions to create a Google Cloud
+Function (e.g. with the [cloudfunctions.serviceAgent Role](https://cloud.google.com/functions/docs/concepts/iam))
+and to create a Google Cloud Scheduler (e.g. with the
+[Cloud Scheduler Job Runner Role](https://cloud.google.com/iam/docs/understanding-roles)).
+
+Here is how to create a scheduled pipeline in ZenML:
+
+```python
+from zenml.config.schedule import Schedule
+
+# Run a pipeline every 5th minute
+# Cron expression generated on `https://crontab.guru`
```

Comment:
Is this here on purpose for people that don't know cron schedules and aren't able to google it?

Reply:
Yes

Reply:
I would suggest making it explicit and providing a clickable link outside of the code box then